app/components/MOBNumberFormat/MOBNumberFormat.view.tsx

```diff
@@ -12,11 +12,20 @@ import { MOBNumberFormatProps } from './MOBNumberFormat';
 // word... The real reason is I have to debug seperating the display value from)
 // the calculated value. This would simplify the matter). So, until we have
 // standardized all values are pico-mob with flex display, this is what we got.
+// USDM comes in micro precison, rather than pico precision
 const MOBNumberFormat = (props: MOBNumberFormatProps): JSX.Element => {
   const { inputRef, onChange, name, value, valueUnit, ...rest } = props;
 
-  const parsedValue =
-    valueUnit === 'pMOB' && typeof value === 'string' ? convertPicoMobStringToMob(value) : value;
+  const decimalScale = valueUnit === 'MOB' || valueUnit === 'pMOB' ? 12 : 6;
```

Comment:
The scale relationship between `MOB` and `pMOB`, as well as `USDM` and `mUSDM` should be an exported constant somewhere. Given that I'd expect these to grow, I might suggest something like
```
export type Token = {
    symbol: string;
    precision: number;
}
export const Tokens: Record<string, Token> = {
    MOB: {
        symbol: "MOB",
        precision: 1000000000000,
    },
    pMOB: {
        symbol: "pMOB",
        precision: 1000000,
    },
    ...
}
```
and then exporting it. The precision is being redefined in many places, and feels like it will incur bugs at some point.

Reply:
good idea!